api/src/main/resources/messages.properties

```diff
@@ -228,5 +228,27 @@ adminui.app.configureMetadata.label=Configure Metadata
 adminui.configuremetadata.adminGroups.description=Extension point for the groupings on the Configure Metadata page
```

Comment:
This pull request is invalid, when a pull request has been merged, you can never commit anymore to your original branch linked to it because it stale at that point as you can see from this pull request